iceoryx_posh/source/runtime/posh_runtime.cpp

```diff
@@ -460,32 +460,32 @@ popo::InterfacePortData* PoshRuntime::getMiddlewareInterface(const capro::Interf
     return nullptr;
 }
 
-RunnableData* PoshRuntime::createRunnable(const RunnableProperty& runnableProperty) noexcept
+NodeData* PoshRuntime::createNode(const NodeProperty& nodeProperty) noexcept
 {
     MqMessage sendBuffer;
-    sendBuffer << mqMessageTypeToString(MqMessageType::CREATE_RUNNABLE) << m_appName
-               << static_cast<cxx::Serialization>(runnableProperty).toString();
+    sendBuffer << mqMessageTypeToString(MqMessageType::CREATE_NODE) << m_appName
+               << static_cast<cxx::Serialization>(nodeProperty).toString();
 
     MqMessage receiveBuffer;
 
     if (sendRequestToRouDi(sendBuffer, receiveBuffer) && (3 == receiveBuffer.getNumberOfElements()))
     {
         std::string mqMessage = receiveBuffer.getElementAtIndex(0);
 
-        if (stringToMqMessageType(mqMessage.c_str()) == MqMessageType::CREATE_RUNNABLE_ACK)
+        if (stringToMqMessageType(mqMessage.c_str()) == MqMessageType::CREATE_NODE_ACK)
         {
             RelativePointer::id_t segmentId;
             cxx::convert::fromString(receiveBuffer.getElementAtIndex(2).c_str(), segmentId);
             RelativePointer::offset_t offset;
             cxx::convert::fromString(receiveBuffer.getElementAtIndex(1).c_str(), offset);
             auto ptr = RelativePointer::getPtr(segmentId, offset);
-            return reinterpret_cast<RunnableData*>(ptr);
+            return reinterpret_cast<NodeData*>(ptr);
         }
     }
 
-    LogError() << "Create runnable got wrong response from message queue :'" << receiveBuffer.getMessage() << "'";
+    LogError() << "Create node got wrong response from message queue :'" << receiveBuffer.getMessage() << "'";
```

Comment:
The message can maybe a little bit more explicit. My proposal "Got wrong response from RouDi while creating node:" << receivebuffer...." This removes also MessageQueue because the IPC channel to RouDi will maybe change later.